src/app/views/Editor/AskForHelp.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+  <div
+    id="ask-for-help"
+    class="z-20"
+  >
+    <s-card
+      v-show="showAskForHelp"
+      :type="type"
+      :style="{
+        marginTop: `${pos.relativeTop + pos.editorTop - marginTopAdjustment}px`,
```

Comment:
```suggestion
        top: `${pos.relativeTop + pos.editorTop - marginTopAdjustment}px`,
```

replace by top so we can click on the existing errors and interract with them, marginTopAdjustement may not be needed then